### PR TITLE
Fix typos in help files

### PR DESCRIPTION
**Pull request by balajikris**
The content to be added to Changelog is as follows:
* fix typographic errors in help files

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@balajikris__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (balajik). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by balajikris** [comment]
while setting up the xplat-cli repo, `node bin/azure --gen` touched these files and the changes seem to be fixing typographical errors. I reviewed the change and it looked alright, I presume the help generator was updated but the updated help files were never pushed back to the repo. Taking this chance to send out a trivial PR, testing if my credentials work on the repo and to get accustomed to the repo.

File: lib/plugins.arm.json [hunk 1]
```diff
@@ -21548,7 +21548,7 @@
               "bool": true,
               "short": "-t",
               "long": "--tier",
-              "description": "tier of the App Service Plan\n\teg:\n\t\t Free, Shared,\n\t\t B1 (Basic Small),    B2 (Basic Medium),    B3 (Basic Large),\n\t\t S1 (Standard Small), S2 (Standard Medium), S3 (Standard Large),\n\t\t P1 (Premium Small),  P2 (Premium Medium),  P3 (Premium Large)\n"
+              "description": "tier of the App Service Plan\n\teg:\n\t\t Free, Shared,\n\t\t B1 (Basic Small),    B2 (Basic Medium),    B3 (Basic Large),\n\t\t P1 (Premium Small),  P2 (Premium Medium),  P3 (Premium Large)\n"
```
Comment (balajikris) [inline lib/plugins.arm.json]: this seems to be a bit more than typo change.

**Comment by balajikris** [comment]
@amarzavery @mayurid : I made a trivial first PR, let me know if I'm good to merge this or if this shouldn't be done for some reason.

**Pull request merged**